docs/faq.md

```diff
@@ -622,9 +622,7 @@ making changes to code, even for seemingly trivial fixes. If a "safe" fix breaks
 
 ## How can I disable Ruff's color output?
 
-Ruff's color output is powered by the [`colored`](https://crates.io/crates/colored) crate, which
-attempts to automatically detect whether the output stream supports color. However, you can force
-colors off by setting the `NO_COLOR` environment variable to any value (e.g., `NO_COLOR=1`).
-
-[`colored`](https://crates.io/crates/colored) also supports the `CLICOLOR` and `CLICOLOR_FORCE`
-environment variables (see the [spec](https://bixense.com/clicolors/)).
+Ruff's color output is powered by the [`owo-colors`](https://crates.io/crates/owo-colors) and
+[`anstream`](https://crates.io/crates/anstream) crates, which attempt to automatically detect
+whether the output stream supports color. However, you can force colors off by setting the
+`NO_COLOR` environment variable to any value (e.g., `NO_COLOR=1`).
```

Comment:
I think `anstream` also supports `CLICOLOR` and `CLICOLOR_FORCE`: https://github.com/rust-cli/anstyle/blob/ed1a598cc2c3b84483c6058a982d045f7f7aecfc/crates/anstyle-query/src/lib.rs#L34

Reply:
Oh good point, I'll restore that paragraph.

Reply:
Honestly had to dig through the source to know for sure (as you can tell)!